### PR TITLE
docs(windows): update screenshots and documentation for Keyman for Windows config

### PR DESCRIPTION
**Pull request by rc-swag**
Fixes: #7966 

This PR updates the version 15.0 screenshots to version 16.0 of the Keyman for Windows Configuration. It also adds some additional screenshots as well as updating some of the documentation to match.

@keymanapp-test-bot  skip

### CODE REVIEW

File: windows/src/desktop/help/basic/config/keyboards.md [hunk 1]
```diff
@@ -40,18 +40,44 @@ Keyman Configuration, see: [Keyboard Task - Enable or Disable a Keyboard](../ena
 
 ## Setting a Hotkey
 
-To set a hotkey for a Keyman keyboard, use the [Hotkeys tab](hotkeys).
+The Hotkey for selecting a keyboard will appear to hte right of the keyboard name.
```
Comment (ermshiperete) [inline windows/src/desktop/help/basic/config/keyboards.md:43]:
```suggestion
The Hotkey for selecting a keyboard will appear to the right of the keyboard name.
```

File: windows/src/desktop/help/basic/enable-or-disable-keyboard.md [hunk 2]
```diff
@@ -33,12 +33,17 @@ To disable a Keyman keyboard:
...
-6.  Click OK.
+6.  Click on the Disable button.
 
-When a keyboard layout is disabled, its name appears in grey and its
-checkbox is unticked.
+    ![](../desktop_images/tab-layout-disable.png)
+
+7.  Click OK.
+
+When a keyboard layout is disabled, its name and details appears and in grey and the disable
```
Comment (ermshiperete) [inline windows/src/desktop/help/basic/enable-or-disable-keyboard.md:44]:
```suggestion
When a keyboard layout is disabled, its name and details appear in grey and the disable
```

**Review by ermshiperete — APPROVED** [review]

File: windows/src/desktop/help/basic/config/keyboards.md [hunk 3]
```diff
@@ -60,23 +86,32 @@ Keyman Configuration:
...
-    (![](../../desktop_images/keyboards-downarrow.png)) beside the keyboard
-    name. The keyboard information window will expand.
-
-    ![](../../desktop_images/keyboards-hotkey.png)
-
-3.  In the information window you can view keyboard details, including:
-
+2.  Select the keyboard name from the keyboard list to expand options associated with it. 
+    The keyboard information window will expand.
+    (![](../../desktop_images/tab-layout-detail1.png))
```
Comment (darcywong00) [inline windows/src/desktop/help/basic/config/keyboards.md:91]:
```suggestion
    ![](../../desktop_images/tab-layout-detail1.png)
```

The outer set of parens are getting rendered on the page.

File: windows/src/desktop/help/basic/config/keyboards.md [hunk 4]
```diff
@@ -60,23 +86,32 @@ Keyman Configuration:
...
+    
+      -   Languages associated with the keyboard 
+      -   Keyboard package name
+      -   Keyboard package version number
+      -   Copyright details
+        
+
+  
+3. Click on the down arrow
+    (![](../../desktop_images/keyboards-downarrow.png)) at the end of the list will reveal
```
Comment (darcywong00) [inline windows/src/desktop/help/basic/config/keyboards.md:102]:
```suggestion
    ![](../../desktop_images/keyboards-downarrow.png) at the end of the list will reveal
```

Depends if you're wanting parens around the down arrow image.

File: windows/src/desktop/help/basic/uninstall-keyboard.md [hunk 5]
```diff
@@ -23,13 +23,15 @@ To uninstall a Keyman keyboard:
...
+    expand options associated with it.
+    
+6.  Click the Uninstall button.
 
-6.  A message box is displayed asking you to confirm this is the package
+7.  A message box is displayed asking you to confirm this is the package
     or keyboard layout that you wish to uninstall. Click OK.
 
-7.  Click OK.
+8.  Click OK.
```
Comment (darcywong00) [inline windows/src/desktop/help/basic/uninstall-keyboard.md:34]: Step 7 ends with "Click OK.". Are there two sets of "OK"?

File: windows/src/desktop/help/basic/config/keyboards.md (hunk 4 above)
Comment (rc-swag) [reply windows/src/desktop/help/basic/config/keyboards.md:102]: Yeah the exiting md that had the "arrow" and "?" had the extra rendered parentheses. Then all references to the keyman icon have no parentheses and just a comma.  I think will remove the parentheses as the icon does not get rendered in the centre so it looks strange.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Review by ermshiperete — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 16.0.132-beta](https://keyman.com/downloads/releases/beta/16.0.132)

**Pull request merged**